packages/example-app-base/src/components/ApiReferencesTable/ApiReferencesTable.tsx

```diff
@@ -459,7 +452,7 @@ export class ApiReferencesTable extends React.Component<IApiReferencesTableProps
     return (
       <>
         {linkTokens.map((token: ILinkToken, index: number) => {
-          let hash: string = '/components/';
+          let hash: string = '/controls/web/';
```

Comment:
@natalieethell I'm pretty sure this is necessary due to the new site's URL structure--let me know if it will cause issues elsewhere